### DISCUSSION TITLE
How to display a payment link on a page

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#127] [General]
Hi,
I use thie shortcode on a page for login user :
[eme_mybookings rsvp_status=1 scope=future template_id=19]
The template_id=19 is 
`<table>
<tbody>
<tr>
<td>#_LOCATIONIMAGETHUMB</td>
<td>Vous avez #_RESPSEATS place(s) réservée(s) pour #_LINKEDNAME du #_STARTDATE au #_ENDDATE.
Paiement par carte : #_FIELD{3}
Déjà payé ? #_PAID
avec #_CONTACTNAME sur le voilier #_LOCATIONNAME situé au #_FIELD{6} : pour annuler cliquer ici #_CANCEL_OWN_LINK
L'ensemble des coéquipiers inscrits sont : #_ATTENDEES</td>
</tr>
</tbody>
</table>``
Is there a placehoders that i can use to add a link for the payment in this template ?

**Comment by liedekef** [comment]
See the booking placeholders at https://www.e-dynamics.be/wordpress/category/documentation/7-placeholders/7-4-rsvp-mails/ : #_PAYMENT_URL

**Comment by f14m07** [comment]
Yes, I use it in an e-mail and it works fine. But I would like to put this link on a page for a login user in the list of his bookings. This placeholder doesn't work on a page

**Reply by liedekef** [reply]
That seems weird ... I'll test that, but normally all booking placeholders can be used in the eme_mybookings template

**Reply by liedekef** [reply]
#_PAYMENT_URL works as expected in the eme_mybookings shortcode/template.

**Comment by f14m07** [comment]
difficult to understand. It didn't work locally. But on my production site it worked. Back on my local site I put the link between `<a href...> `and it worked. A bit of a mystery to me. Sorry I made you work for nothing. Thanks for your help

**Reply by liedekef** [reply]
#_PAYMENT_URL is "just" the link, to make it clickable you need to enclose it in a a-tag. Some themes might convert this for you (as do mail clients) but to make sure the link is clickable always enclose it.
